tests/shell/shell_recorder/shell_recorder.sh

```diff
@@ -57,12 +57,12 @@ execute()
 
 	if [ -n "$RETCMP" ];
 	then
-		nbTest=$(( nbTest + 1 ))
+		tests=$(( tests + 1 ))
```

Comment:
The name "nbTest" is also used in run_all and the cframework. errors and tests are really generic words which I would rather avoid as variable name. (Makes grepping and search&replace more difficult.)